### ISSUE TITLE
Saving twice in a row often breaks the watcher

### ISSUE THREAD

**Issue by jgw96** [driftyco/ionic-app-scripts#457]
_From @larssn on November 22, 2016 11:35_

**Ionic version:**  (check one with "x")
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[x] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
When saving multiple times in short succession, the watcher apparently breaks, and webpack gets stuck in "started" mode.

```
[12:28:29]  build started ... 
[12:28:29]  transpile update started ... 
[12:28:30]  transpile update finished in 1.78 s 
[12:28:30]  webpack update started ... 
[12:28:37]  build started ... 
[12:28:42]  transpile update started ... 
[12:28:42]  transpile update finished in 367 ms 
[12:28:42]  webpack update started ... 
[12:28:51]  webpack update finished in 8.74 s 
[12:28:53]  build finished in 24.70 s  <------ PARALLEL BUILDS

[12:28:53]  build finished in 16.36 s  <------ PARALLEL BUILDS

[12:29:20]  build started ... 
[12:29:20]  transpile update started ... 
[12:29:20]  transpile update finished in 370 ms 
[12:29:20]  webpack started ...  <--------- NEVER FINISHES
```

**Expected behavior:**
I'd expect that when I hit save, the watcher RESTARTS, instead of doing what appears to be a parallel build.

**Steps to reproduce:**
Save twice within a second or two.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
Cordova CLI: 6.4.0 
Ionic Framework Version: 2.0.0-rc.3
Ionic CLI Version: 2.1.8
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.46
ios-deploy version: 1.9.0 
ios-sim version: 5.0.11 
OS: macOS Sierra
Node Version: v7.1.0
Xcode version: Xcode 8.1 Build version 8B62
```

_Copied from original issue: driftyco/ionic#9281_

**Comment by sgireddy** [comment]
Any solution to this issue? I had auto save enabled in my VS Code triggering this issue (I am going to disable as a workaround). 

The worst part is that I had to use "Ctrl + Z" then kill running node process before I can run ionic serve.

**Comment by sgireddy** [comment]
BTW, is there a way to stop ionic-cli gracefully? looks it "q" or "quit" doesn't work anymore.

**Comment by jayeshanandani** [comment]
Can you please try with `0.0.48` ionic app scripts? I have not noticed this in latest versions.

**Comment by larssn** [comment]
I am already using 0.0.48, and the problem stubbornly persists.

**Comment by jayeshanandani** [comment]
@larssn: can you please update ionic-cli to `2.1.18` with `npm install ionic@latest -g` and try again. And in addition if you can try to use LTS version of node `6.9.2` and recheck the serve again.

**Comment by cmaart** [comment]
Experiencing same issues on 1.0.0

**Comment by DrMabuse23** [comment]
Hi same probs:
```
[16:25:31]  build started ... 
[16:25:31]  transpile update started ... 
[16:25:32]  transpile update finished in 253 ms 
[16:25:32]  webpack update started ... 
[16:25:42]  build started ... 
[16:25:42]  transpile update started ... 
[16:25:42]  transpile update finished in 184 ms 
[16:25:42]  webpack update started ... 
[16:25:51]  build started ... 
[16:25:51]  transpile update started ... 
[16:25:51]  transpile update finished in 155 ms 
[16:25:51]  webpack update started ... 
[16:25:52]  webpack update finished in 1.33 s 
[16:25:52]  build finished in 21.12 s 

[16:25:52]  build finished in 10.55 s 

[16:26:09]  build finished in 18.19 s 
```
info : 

Cordova CLI: 6.5.0 
Ionic Framework Version: 2.0.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.0.0
ios-deploy version: 1.8.3 
ios-sim version: 5.0.3 
OS: macOS Sierra
Node Version: v6.7.0
Xcode version: Xcode 8.2.1 Build version 8C1002

**Comment by danbucholtz** [comment]
Dan note: Just queue the build if another one is active. Should be simple.

**Comment by danbucholtz** [comment]
This is fixed in `1.2.0`.

Thanks,
Dan
